backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/TaskServiceInMemory.java

```diff
@@ -28,6 +28,15 @@ public String submit(Task task) {
     return task.getId();
   }
 
+  @Override
+  public String submitTaskFromName(String name, String userName, String parameters) {
+    throw new UnsupportedOperationException("Not supported when using in memory task service");
+  }
+
+  public String submitTaskFromName(String name) {
+    return this.submitTaskFromName(name, null, null);
```

Comment:
overload method instead of passing null , usually does not end well

Reply:
this is the overloaded method right?